account_financial_report/wizard/aged_partner_balance_wizard.py

```diff
@@ -60,8 +60,10 @@ def onchange_company_id(self):
             res['domain']['account_ids'] += [
                 ('company_id', '=', self.company_id.id)]
             res['domain']['partner_ids'] += [
+                '&',
                 '|', ('company_id', '=', self.company_id.id),
-                ('company_id', '=', False)]
+                ('company_id', '=', False),
+                ('parent_id', '=', False)]
```

Comment:
This part of the domain is not correct at all: you can have subsidiaries marked as company (`is_company`), instead of individual, having a parent_id.  See the computation of `commercial_parent_id` as reference:

https://github.com/odoo/odoo/blob/3c45f08825b1a0de35f2027f7cb53977f2c11f08/odoo/addons/base/models/res_partner.py#L276

A possible domain would be:
```
'|',
('parent_id', '=', False),
('is_company', '=', True),
```

Reply:
Done